cli/Cargo.toml

```diff
@@ -12,23 +12,25 @@ keywords = ["crypto", "blockchain", "ledger"]
 categories = ["cryptography::cryptocurrencies"]
 
 [features]
+default = ["bridge", "telemetry", "schema-endpoint"]
+
 bridge = ["iroha_core/bridge"]
```

Comment:
I would add a few comments here. 
```
# Interoperability with popular blockchain networks (Substrate, Ether etc.)

# Support for Decentralised EXchange use cases. Includes functionality necessary for the functioning of atomic exchange instruction. 

# Support for lightweight telemetry, including diagnostics

# Support for developer-specific telemetry. Should not be enabled on production builds. 

# Support for returning the schema for use in SDKs from the local build. Mostly used for debugging. 

# Support for internal testing infrastructure necessary for integration tests. Disable in production. 

```

Reply:
I recently opened an [issue](https://github.com/hyperledger/iroha/issues/2113) for this